meetings/2023-06-07.md

```diff
@@ -72,21 +72,24 @@ URL: https://www.w3.org/2023/09/TPAC/
 * LD: What' the status of HttpSig specification?
-* HS: HTTP Signatures in IETF. Last call was in September. HTTPSig is a lot more Solid now.
-* HS: Written in Scala that compiles to JavaScript.
+* HS: [HTTP Signatures](https://datatracker.ietf.org/doc/draft-ietf-httpbis-message-signatures/) in being developed in the IETF HTTPBis group. The authors were in last-call in September at version 13. The received enough feedback to get them to version 17 in the meantime. But it is close to finished. HTTPSig is a lot more Solid now.
+* HS: [My implementation of httpSig is on github](https://github.com/bblfish/httpSig) , It is written in Scala that compiles to JavaScript, so that I can be used on the client and on the server. (It would need work for nodeJS) 
 * LD: Is it for any authenticated agent? ACLs working for keyid? See it working with WebID?
-* HS: It is declared logically. Could still use OIDC, because `401` can come back to use that. Can use passwords too. The client should have ??? the wallet that I'm writing. It should be able to select which of these descriptions fits for authentication needs.
-* TBL: Good demo. I suppose ??? How much would it take to config option for CSS to use it? You got Scala→JavaScript. CSS needs JS. Is there a test suite?
-* HS: A lot of tests for HTTPSig. This is the beginning of the test — the data I should be getting back... If you send me someone that's a JavaScript developer to help me to help them (???), we can find the right people. I don't know CSS that well. I know people that would know how to bridge that gap to do it. I'd be happy with more people using and maintaining.
+* HS: It fits in nicely with other HTTP auth schemes. A server could both support HTTP Sig, OIDC, or passwords too. (and even WebID-TLS). The client should have a Solid Wallet (an [EU project](https://nlnet.nl/project/SolidWallet/)) with a number of credentials for the user (passwords, keys, etc...) It should be able to select which of the ACL descriptions fit which keys and decide which is best for the authentication needs of the user.
+* TBL: Good demo. I suppose ??? How much would it take for CSS to use it? You got Scala→JavaScript. CSS needs JS. Is there a test suite?
+* HS: HttpSig depends on [RFC 8941: Structured Header Fields](https://www.rfc-editor.org/rfc/rfc8941.html) for which I have an implementation [rfc8941.scala](https://github.com/bblfish/httpSig/blob/main/rfc8941/shared/src/main/scala/run/cosy/http/headers/Rfc8941.scala), and a lot of [detailed tests](https://github.com/bblfish/httpSig/blob/main/rfc8941/shared/src/test/scala/run/cosy/http/headers/Rfc8941_Test.scala). That is used by what I call [ietfSig](https://github.com/bblfish/httpSig/tree/main/ietfSig) (as the RFC does not yet have a number), and there are alos a lot of tests for the HTTPSig, which I tried to keep generic so that they can work across platforms. (The current platforms are [akka](https://akka.io) (Java( and [http4s](https://http4s.org) (Java and JS)).  
```

Comment:
```suggestion
* HS: HttpSig depends on [RFC 8941: Structured Header Fields](https://www.rfc-editor.org/rfc/rfc8941.html) for which I have an implementation [rfc8941.scala](https://github.com/bblfish/httpSig/blob/main/rfc8941/shared/src/main/scala/run/cosy/http/headers/Rfc8941.scala), and a lot of [detailed tests](https://github.com/bblfish/httpSig/blob/main/rfc8941/shared/src/test/scala/run/cosy/http/headers/Rfc8941_Test.scala). That is used by what I call [ietfSig](https://github.com/bblfish/httpSig/tree/main/ietfSig) (as the RFC does not yet have a number), and there are also a lot of tests for the HTTPSig, which I tried to keep generic so that they can work across platforms. (The current platforms are [akka](https://akka.io) (Java and [http4s](https://http4s.org) (Java and JS)).  
```